project/templates/configs/springboard.ini

```diff
@@ -8,7 +8,9 @@ pyramid.debug_routematch = false
 
 pyramid.default_locale_name = {{default_language}}
 
-unicore.content_repo_urls = {{ucd_host}}/repos/{{repo_name}}.json
+unicore.content_repo_urls =
```

Comment:
I think `springboard-iogt` might have 2 of this settings e.g `unicore.content_repo_urls` and `unicore.own_content_repo_url`